src/components/WebexInterstitialMeeting/WebexInterstitialMeeting.jsx

```diff
@@ -19,13 +21,17 @@ import WebexMeetingInfo from '../WebexMeetingInfo/WebexMeetingInfo';
  */
 export default function WebexInterstitialMeeting({className, meetingID, style}) {
   const cssClasses = webexComponentClasses('interstitial-meeting', className);
+  const {localVideo} = useMeeting(meetingID);
 
   return (
     <div className={cssClasses} style={style}>
       {meetingID ? (
         <>
           <WebexMeetingInfo className="interstitial-meeting-info" meetingID={meetingID} />
-          <WebexLocalMedia className="interstitial-media" banner="My preview" meetingID={meetingID} mediaType="video" />
+          <div className="interstitial-media-container">
+            <WebexLocalMedia className="interstitial-media" meetingID={meetingID} mediaType="video" />
+            {localVideo.stream && <Banner className="preview-banner">My preview</Banner>}
```

Comment:
is the class on Banner still needed?

Reply:
yes, to know when to display the themed banner and when to display the old banner